src/Perception/include/BipedalLocomotion/Perception/Features/PointCloudProcessor.h

```diff
@@ -0,0 +1,356 @@
+/**
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_PERCEPTION_PCL_PROCESSOR_H
+#define BIPEDAL_LOCOMOTION_PERCEPTION_PCL_PROCESSOR_H
+
+#include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
+#include <BipedalLocomotion/TextLogging/Logger.h>
+
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+#include <pcl/filters/voxel_grid.h>
+#include <pcl/filters/statistical_outlier_removal.h>
+#include <pcl/kdtree/kdtree.h>
+#include <pcl/segmentation/extract_clusters.h>
+#include <pcl/common/transforms.h>
+
+namespace BipedalLocomotion {
+
+namespace Perception {
+
+namespace Features {
+
+
+struct PCLProcessorParams
```

Comment:
I think you can expand ``Params`` to ``Parameters``.

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/236/commits/4ff3d088e4c925a306cb78dac646884df81cbeda